fix: ROOT-17: Check conditions for COCO/YOLO Keypoints export

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/15255468650)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/467
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15254239625

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15255468650

triklozoid commented:
/git merge
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/15255913195)
> Successfully merged:  create mode 100644 docs/themes/v2/source/images/import-export/keypoints.png

triklozoid committed:
Merge branch 'develop' into 'fb-ROOT-17'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/15255913195

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/15256991524

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-ROOT-17'

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/15256991524
